example_packages/compiled_library/minpack/fpm.toml

```diff
@@ -0,0 +1,9 @@
+name = "minpack"
+
+[library]
+include-dir = ["inc"]
+library-dir = ["src"]
+
+[build]
+link = "minpack"
+external-modules = ["minpack_module"]
```

Comment:
These four features allow `fpm` to distribute binary link libraries, like `minpack`.